sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "2.1.0.dev"
+__version__ = "1.28.2"
```

Comment:
Wrong aswell

Reply:
I think we should skip packages that are released individually no?

Reply:
Yep, looks like the script updating this is also forgetting about `exporter/`, `propagator/`and `processor/`?

Reply:
> I think we should skip packages that are released individually no?

I think @lzchen mentioned this, I will make a note of it. IIUC it should be inconsequential because the independent release packages won't be uploaded during the release workflow. WDYT?

> looks like the script updating this is also forgetting about `exporter/`, `propagator/`and `processor/` and maybe other dirs?

Ahh that's more concerning, I'll take a look thanks!

Reply:
> > looks like the script updating this is also forgetting about `exporter/`, `propagator/`and `processor/` and maybe other dirs?
> 
> Ahh that's more concerning, I'll take a look thanks!

I guess we're missing a ton of entries for these in `eachdist.ini`?

Reply:
PTAL at the Prepare release PR: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3017